index.js

```diff
@@ -19,6 +19,35 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    const myRecipe = await Recipe.create({
```

Comment:
Everything is correct and works here, however, it would be a better practice if you stored your `myRecipe` object in a variable, and then, do the `Recipe.create(myRecipe)` method.